src/engine/client/cl_console.cpp

```diff
@@ -351,8 +351,8 @@ bool Con_CheckResize()
 					}
 					if ( len > consoleState.textWidthInChars )
 					{
-						consoleState.lines.emplace_back( begin, token.Begin() );
-						begin = token.Begin();
+						consoleState.lines.emplace_back( begin, token.RawToken().begin() );
```

Comment:
I'm surprised that this doesn't want normalizedtoken

Reply:
is this something to be fixed in grep rather than worked around in your token iterator API?